### PR TITLE
Add statsd integration to breeze

### PR DESCRIPTION
**Pull request by turbaszek**
---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).

### CODE REVIEW

File: scripts/ci/docker-compose/integration-statsd.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+---
+version: "2.2"
+services:
+  statsd-exporter:
+    image: prom/statsd-exporter
```
Comment (potiuk) [inline scripts/ci/docker-compose/integration-statsd.yml]: We are already using airffow''s version of the statsd exporter in the helm chart and it is published in airflow dockerhub published https://github.com/apache/airflow/blob/bb00f164da6c84f078e66582217832aeedb2a966/chart/values.yaml#L136 

We also have our own way to build and publish it as needed (it is super-easy): https://github.com/apache/airflow/blob/master/chart/dockerfiles/statsd-exporter/build_and_push.sh

Shoud we use that version instead? Should we rebuild and push a new version instead?

Comment (turbaszek) [reply scripts/ci/docker-compose/integration-statsd.yml]: Sure, didn't know that!

**Comment by jhtimmins** [comment]
@turbaszek @potiuk Going back through older PRs. Is this still in progress?

**Comment by turbaszek** [comment]
@jhtimmins I updated the PR

**Review by potiuk — APPROVED** [review]

**Comment by potiuk** [comment]
MySQL and K8S failures are unrelated

**Pull request merged**